### PR TITLE
Refreshed README For 270 Clamping Hinges

### PR DESCRIPTION
**Pull request by Alexander-T-Moss**
* [✅] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [✅] Folder and file naming:
    * Folders and filenames shouldn't contain spaces. Only letters `a-zA-Z`, numbers `0-9`, underscores `_`, hyphens `-` and periods `.`
    * Primary color: `part_xyz.stl`
    * Accent color: `[a]_part_xyz.stl`
    * Opaque color (Blocks light): `[o]_part_xyz.stl`
    * Clear/transparent color (Allows light): `[c]_part_xyz.stl`
    * Quantity, if more than one is needed: `part_xyz_x4.stl`
  * [✅] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [✅] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [✅] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [✅] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [✅] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [✅] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [✅] Submitted STL files are oriented and scaled properly for printing.
  * [✅] Submission includes a CAD file in the form of a `.STEP` or `.SCAD` file
  * [✅] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?

- Nothing added, just refreshing existing mod - 270 clamping hinges

#### How was it tested? 

- No print files were changed other than moved into different folders or renamed so all still works like before

#### Any background context you want to provide?

- Building on comments and feedback

#### Screenshots (if appropriate)

- N/a

#### Further notes

- N/a

### CODE REVIEW

File: printer_mods/AlexanderT-Moss/270-Clamping-Hinges/README.md [hunk 1]
```diff
@@ -1,70 +1,68 @@
...
- At the moment I have only tested using 3mm thick foam and that works well, you may be able to get away with 4mm foam, but for anything thicker, you'll need to modify part of the hinge to accommodate this.
- 
- ### Keeping The Doors Shut ###
- 
- I am currently using clamps designed by [_v6cl_](https://github.com/v6cl/My-Voron2.4-Customs/tree/main/Panel_Locker)
 
- ### Different Styles Of Face Plates ###
+ ## Do I Need Foam Tape? ##
+
+ **YES**, You will need to use **3mm foam tape** with these hinges - if you wearn't using foam tape before see this as the right time to add foam tape as you should be using it anyway :)
```
Comment (MapleLeafMakers) [inline printer_mods/AlexanderT-Moss/270-Clamping-Hinges/README.md:23]: `wearn't` should be `weren't`

File: printer_mods/AlexanderT-Moss/270-Clamping-Hinges/README.md [hunk 2]
```diff
@@ -1,70 +1,68 @@
...
- :-------------------------------:|:-------------------------------:|:-------------------------------:
- ![Logo-Variation-0.png](https://github.com/Alexander-T-Moss/VoronUsers/blob/master/printer_mods/AlexanderT-Moss/270-Clamping-Hinges/Images/Logo-Variation-0.png?raw=true)         |  ![Logo-Variation-1.png](https://github.com/Alexander-T-Moss/VoronUsers/blob/master/printer_mods/AlexanderT-Moss/270-Clamping-Hinges/Images/Logo-Variation-1.png?raw=true)  | ![Logo-Variation-2](https://github.com/Alexander-T-Moss/VoronUsers/blob/master/printer_mods/AlexanderT-Moss/270-Clamping-Hinges/Images/Logo-Variation-2.png?raw=true)
+  ## Is The Clamping Force Strong Enough? ##
+
+  If you print these with any half decent filament and your **panels are 3mm thick** then the clamping force is more than sufficiently strong. When printing, the panels are additionaly held in place securly by the door locking clamps linked above. You may find opening the doors quickly will make them shift slightly but the panels can be alligned again by hand quite quickly - I recommend closing the panels together at the same time to help allign them.
```
Comment (MapleLeafMakers) [inline printer_mods/AlexanderT-Moss/270-Clamping-Hinges/README.md:33]: `additionaly` should be `additionally`
`securly` should be `securely`
`alligned` should be `aligned`
`allign` should be `align`

**Comment by Alexander-T-Moss** [comment]
Okay, fixed all the typos in the readme :)

**Comment by Alexander-T-Moss** [comment]
It's been a while... but is it possible for this PR to be resolved and merged?

**Comment by FHeilmann** [comment]
Hi, 

I want to thank you for your submission and want to personally apologize that your submission has not gotten the review it deserves.
We are in the process to restructure the repo to streamline reviews and hopefully solve this issue in the future.

Unfortunately, this means that we will require you to resubmit the mod under the new system.

Please check [here](https://github.com/VoronDesign/VoronUsers/wiki/VoronUsers-Overhaul) for details and instructions on how to proceed. 
If you have any questions, please reach out in the #voronuser_mods channel on the VoronDesign discord.
If you have since moved on from this mod and don't want to resubmit, no further action is necessary from your side.

Bests,

Florian a.k.a. Haribro

**Pull request closed without merging**